source/include/BuildConfig.h

```diff
@@ -28,6 +28,10 @@ SDG
 	#define kVireoOS_windows 1
 #endif
 
+#ifdef _MSC_VER
```

Comment:
Close to the bottom of the file, there is already a Vireo windows section, shouldn't be put it there?

 #elif (kVireoOS_windows)
 #if (_MSC_VER < 1900)		 #if (_MSC_VER < 1900)
     #define snprintf _snprintf		     #define snprintf _snprintf
 #endif		 #endif